src/lib.rs

```diff
@@ -116,16 +121,23 @@ impl<T: Translation + Clone> Mapping<T> {
         }
     }
 
+    /// Returns whether this mapping is currently active.
+    pub fn active(&self) -> bool {
+        self.previous_ttbr.is_some()
+    }
+
     /// Activates the page table by setting `TTBRn_EL1` to point to it, and saves the previous value
     /// of `TTBRn_EL1` so that it may later be restored by [`deactivate`](Self::deactivate).
     ///
     /// Panics if a previous value of `TTBRn_EL1` is already saved and not yet used by a call to
     /// `deactivate`.
```

Comment:
It might be worth mentioning here that this is a no-op (apart from keeping track of state) on non-aarch64 targets.